policyengine_us/parameters/gov/states/ny/tax/income/credits/ctc/additional_number_of_children.yaml

```diff
@@ -0,0 +1,10 @@
+description: Minimum number of children eligible for additional child tax credit. 
+values:
+  2018-01-01: 3
+metadata:
+  unit: int
+  period: year
+  label: NY CTC additional tax credit children amount
+  reference:
+    - title: Instructions for Form IT-213 Claim for Empire State Child Credit
```

Comment:
could you add 2021, 2022 , and 2023 tax forms as well as the legal code here?

Reply:
> could you add 2021, 2022 , and 2023 tax forms as well as the legal code here?

cannot find any limit in the legal code

Reply:
Also, starting from 2023, no more min age limitation. And no worksheet B anymore. in this case we may need to create new calculations, please advice

Reply:
If there are structural changes for 2023 lets keep this PR to 2021/2022 structure and file a new issue to re-adjust this credit to account for 2023 changes, I would rather make sure that we have modeled the base correctly as it will be easier to add the 2023 conditions afterwards